### PR TITLE
Added masonite upgrade to version 4

### PR DESCRIPTION
**Pull request by josephmancuso**

### CODE REVIEW

**Comment by waghanza** [comment]
Is the line https://github.com/MasoniteFramework/masonite/blob/c67a06946eab03cebb0b6b45c1236bba41ae1f22/src/masonite/authorization/Gate.py#L2 correct ?

```python
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.9/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/usr/src/app/wsgi.py", line 1, in <module>
    from masonite.foundation import Application, Kernel
  File "/usr/local/lib/python3.9/site-packages/masonite/foundation/__init__.py", line 2, in <module>
    from .Kernel import Kernel
  File "/usr/local/lib/python3.9/site-packages/masonite/foundation/Kernel.py", line 27, in <module>
    from ..tests.HttpTestResponse import HttpTestResponse
  File "/usr/local/lib/python3.9/site-packages/masonite/tests/__init__.py", line 1, in <module>
    from .TestCase import TestCase
  File "/usr/local/lib/python3.9/site-packages/masonite/tests/TestCase.py", line 10, in <module>
    from ..request import Request
  File "/usr/local/lib/python3.9/site-packages/masonite/request/__init__.py", line 1, in <module>
    from .request import Request
  File "/usr/local/lib/python3.9/site-packages/masonite/request/request.py", line 7, in <module>
    from ..authorization import AuthorizesRequest
  File "/usr/local/lib/python3.9/site-packages/masonite/authorization/__init__.py", line 2, in <module>
    from .Gate import Gate
  File "/usr/local/lib/python3.9/site-packages/masonite/authorization/Gate.py", line 2, in <module>
    from masoniteorm import Model
ModuleNotFoundError: No module named 'masoniteorm'
```

**Comment by josephmancuso** [comment]
@waghanza masoniteorm was missing from requirements

**Review by waghanza — CHANGES_REQUESTED** [review]
When requesting `/`, I have the following error
```python
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/masonite/foundation/response_handler.py", line 27, in response_handler
    application.resolve(provider.boot)
  File "/usr/local/lib/python3.9/site-packages/masonite/container/container.py", line 223, in resolve
    return obj(*objects)
  File "/usr/local/lib/python3.9/site-packages/masonite/providers/RouteProvider.py", line 21, in boot
    request = self.application.make("request")
  File "/usr/local/lib/python3.9/site-packages/masonite/container/container.py", line 105, in make
    raise MissingContainerBindingNotFound(
masonite.exceptions.exceptions.MissingContainerBindingNotFound: request key was not found in the container
```

File: python/masonite/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
-masonite>=3.0,<3.1
+masonite>=4.0,<4.1
+masonite-orm
```
[the remark this answers is not in the document]
Comment (josephmancuso) [reply python/masonite/requirements.txt:2]: No. Masonite-or is technically not required from Masonite anymore. Its a standalone ORM that is installed in default projects

[the remark this answers is not in the document]
Comment (waghanza) [reply python/masonite/requirements.txt:2]: what if I do not required any orm layer ? may I remove database configurations ?

[the remark this answers is not in the document]
Comment (josephmancuso) [reply python/masonite/requirements.txt:2]: Yes you can remove any references to a database

[the remark this answers is not in the document]
Comment (josephmancuso) [reply python/masonite/requirements.txt:2]: Actually I just checked the codebase and there is a reference to masoniteorm in the core code so maybe it is required actually

[the remark this answers is not in the document]
Comment (waghanza) [reply python/masonite/requirements.txt:2]: Ok, glad to help any project. I'll leave the reference in here for now

**Comment by waghanza** [comment]
can you help on https://github.com/the-benchmarker/web-frameworks/pull/4809#pullrequestreview-840110666 @josephmancuso ?t

**Comment by josephmancuso** [comment]
review what? This should be closed in favor of #4961

**Pull request closed without merging**